### PR TITLE
W1 - Message Brokers - Adde configuration for the Azure Service Bus

### PR DESCRIPTION
**Pull request by Yariki**
I have introduced the settings to Azure Service Bus. 
The Catalog service has a sending policy.
The Basket service has a listening policy.

### CODE REVIEW

File: src/CartingService/appsettings.json [hunk 1]
```diff
@@ -5,5 +5,8 @@
       "Microsoft.AspNetCore": "Warning"
     }
   },
-  "AllowedHosts": "*"
+  "AllowedHosts": "*",
+  "ServicecBus": {
+    "ConnectionString": "Endpoint=sb://softarch.servicebus.windows.net/;SharedAccessKeyName=ListenPolicy;SharedAccessKey=XkmqgTCWITiApANz5eR/gs96uoK8ux+z/CU5nCddquQ=;EntityPath=product"
```
Comment (rkosmirak) [inline src/CartingService/appsettings.json]: The connection strings should not be added to the source control

File: src/CatalogService/src/CatalogService.Api/appsettings.json [hunk 2]
```diff
@@ -7,7 +7,10 @@
     },
     "AllowedHosts": "*",
     "UseInMemoryDatabase": false,
-    "ConnectionStrings": {
-        "DefaultConnection": "Server=(localdb)\\mssqllocaldb;Database=CatalogService;Trusted_Connection=True;MultipleActiveResultSets=true;"
-    }
+  "ConnectionStrings": {
```
Comment (rkosmirak) [inline src/CatalogService/src/CatalogService.Api/appsettings.json]: The connection strings should not be added to the source control

**Review by rkosmirak — CHANGES_REQUESTED** [review]

File: src/CartingService/appsettings.json (hunk 1 above)
Comment (Yariki) [reply src/CartingService/appsettings.json]: Fixed, moved to secrets.

File: src/CatalogService/src/CatalogService.Api/appsettings.json (hunk 2 above)
Comment (Yariki) [reply src/CatalogService/src/CatalogService.Api/appsettings.json]: Fixed, moved to secrets.

File: src/CartingService/Program.cs [hunk 3]
```diff
@@ -45,6 +49,8 @@
 
 builder.Services.ConfigureOptions<ConfigureSwaggerOptions>();
 
+CheckBusSettings(builder);
```
Comment (rkosmirak) [inline src/CartingService/Program.cs:52]: What is the point of this method?
Please, check this article for the ideas about validation of appsettings https://code-maze.com/aspnet-configuration-options-validation/

Comment (Yariki) [reply src/CartingService/Program.cs:52]: Actually, that was only for me. Just to check if it works :)

**Pull request merged**